bluesky/examples.py

```diff
@@ -523,7 +523,7 @@ def trigger(self):
         return NullStatus()
 
     def read(self):
-        return self._result
+        return super().read()
```

Comment:
@CJ-Wright What? I'm not sure how this is different than just removing the method.